beacon-chain/VALIDATOR_REGISTER.md

```diff
@@ -0,0 +1,35 @@
+# Validator Registration Workflow
+
+This doc will summarize the work flow of registering to become a validator in the beacon chain. The scope is within Ruby Release.
+
+### Step 1: Deploy validator registration contract if it hasn't been done
+To deploy VRC, we can use [deployVRC](https://github.com/terenc3t/geth-sharding/tree/contract-util/contracts/deployVRC) utility.  
+Once we get the VRC contract address, we can launch our beacon chain node
+```
+# Deploy VRC with keystore UTCJSON and password
+go run deployVRC.go --UTCPath /path/to/your/keystore/UTCJSON --passwordFile /path/to/your/password.txt
+# Deploy VRC with private key
+go run deployVRC.go --privKey 8a6db3b30934439c9f71f1fa777019810fd538c9c1e396809bcf9fd5535e20ca
+
+INFO[0039] New contract deployed at 0x559eDab2b5896C2Bc37951325666ed08CD41099d
+```
+
+### Step 2: Launch beacon chain node
+Launch beacon chain node with account holder's public key and the VRC address we just deployed
+```
+./bazel-bin/path/to/your/beacon-chain/binary --vrcaddr 0x527580dd995c0ab81d01f9993eb39166796877a1 --pubkey aaace816cdab194b4bc6c0de3575ccf917a9b9ecfead263720968e0e1b45739c
+
+```
+
+### Step 3: Send a transaction to VRC with 32 ETH and public key corresponded to beacon chain node account holder's public key
+It's up to you on how you want to send the deposit transaction to VRC. I perosnally find Remix to be the easier. With Remix, I sent a transaction to the deposit method with 32 ETH and the same public key we launched our beacon chain node with.
```

Comment:
Typo.

Can you remove first person references like “I personally” since the reader won’t understand who the author is when reading this?